CONTRIBUTING.md

```diff
@@ -41,15 +41,15 @@ To run tests:
 - Clone the repository:
 
   ```console
-  $ git clone git@github.com:zendframework/zend-expressive-fastroute.git
+  $ git clone git://github.com/zendframework/zend-expressive-fastroute.git
```

Comment:
I'm not sure about this. I've already mentioned my concerns at https://github.com/zendframework/zend-stratigility/pull/89#discussion_r97573832